plugins/gear/gear.h

```diff
@@ -15,11 +15,11 @@
 
 #include "qc_plugininterface.h"
 #include <QDialog>
-
-class QPointF;
-class QSpinBox;
-class QDoubleSpinBox;
-class QComboBox;
```

Comment:
The previous code is preferred here, use forward class declaration instead of includes for pointer variables.
The correct place for includes of member pointer classes is the cpp file (where they are already).

Please restore the previous `class Q...;` lines and add `class QCheckBox;` for the new member pointer variables.

Reply:
I used that because I had problems compiling the moc files, but I'll try to put them as they were in the previous code.